ddocs/medic-db/medic/views/contacts_by_depth/map.js

```diff
@@ -10,6 +10,10 @@ function(doc) {
       if (parent._id) {
         emit([parent._id], value);
         emit([parent._id, depth], value);
+        if(doc.type === 'person' && depth > 1){
```

Comment:
I don't think this approach will work. With configurable hierarchies, the `type` won't be `person` even if this is a person contact, and there will be no way to know whether this is a person or not. I think your feature request will need some additional design decisions.

Reply:
Okay thanks @dianabarsan